data/json/monsters.json

```diff
@@ -145,6 +145,115 @@
     "death_function": [ "NORMAL" ],
     "flags": [ "SEES", "HEARS", "SMELLS", "CHITIN", "CLIMBS" ]
   },
+  {
+    "id": "mon_ant_acid",
+    "type": "MONSTER",
+    "name": "giant spitter ant",
+    "description": "A monstrous red ant with a swollen abdomen, that ends with a small orifice at the tip.  Glistening liquid seems to drip out periodically.",
+    "default_faction": "ant",
+    "species": [ "INSECT" ],
```

Comment:
These spitter ants are still part of the same faction as the normal ants? Intentional or not?

Reply:
Wow, just saw this. Yes, this probably should be changed at some point -- they are clearly different species. However, I don't know enough about both species since there are some that are capable of oligogony and complex slave-making behaviors (some being parasitic).

For simplicity, probably red vs brown ant factions.